tests/test_helpers.py

```diff
@@ -56,8 +56,26 @@
     (["helper", "l_contacts", "run", "-n", "ny", "-i", "col", "-o", "coffee", "-d", "2019-01-15", "-r", "2"],
      ""
      ),
+    (["helper", "u_milestone", "20sb_firstprojectum", "--number", "4",
+      "-s", "c", "--due_date", "2022-01-01"],
+     "20sb_firstprojectum has been updated in projecta\n"
+     ),
+    (["helper", "u_milestone", "20sb_firstprojectum"],
+     "Please choose from one of the following to update/add:\n"
+     "1. Kick off meeting    due date: 2020-05-06    finished\n"
+     "2. Kick off meeting    due date: 2020-05-06    finished\n"
```

Comment:
can we have only one kick off meeting?

Reply:
done!
In the schemas we had kickoff and a milestone called kick off meeting. 
I deleted the milestone named kick off meeting. It is why I had to delete a field from tests/outputs/a_projectum/projecta.yaml . 
Is this correct? Or do you want me to not include the kickoff in my list instead?